js/scripts.js

```diff
@@ -0,0 +1,8 @@
+
+var text = 'Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach'; 
+var animal = 'Zielone słonie';
+var animalUpperCased = animal.toUpperCase();
+var replacedAnimal = text.replace('Papugi', animalUpperCased);
```

Comment:
Zrób tak żeby zamiast Papug, zamieniany był zawsze pierwszy wyraz zmiennej `text` - bez względu na to co tam będzie. Możesz użyć do tego metody `split` (google -> "mdn split")